crates/ruff_benchmark/benches/red_knot.rs

```diff
@@ -17,17 +17,17 @@ import typing
 from bar import Bar
 
 class Foo(Bar):
-    def foo() -> str:
+    def foo() -> object:
         return "foo"
 
     @typing.override
-    def bar() -> str:
+    def bar() -> object:
         return "foo_bar"
 "#;
 
 static BAR_CODE: &str = r#"
 class Bar:
-    def bar() -> str:
+    def bar() -> object:
         return "bar"
```

Comment:
This change is because otherwise we panic when trying to resolve the `str` builtin symbol, because the `str` class in typeshed's `builtins.pyi` file has a `Subscript` node as its sole base, and we don't support those yet. An alternative solution would be to use a custom typeshed in the benchmark here -- but then the benchmarks won't catch any unexpected performance regressions caused by a PR syncing our vendored typeshed stubs.

Reply:
Yeah, I think this is the best approach for now; we should be adding to our benchmark anyway as we add new supported features.

Reply:
Can we land this change separately, then rebase so that we get proper benchmark numbers.

Reply:
> Can we land this change separately, then rebase so that we get proper benchmark numbers.

sure

Reply:
Oh I just did https://github.com/astral-sh/ruff/pull/12393 and already rebased this PR on top of it